src/dev/Rule/EnforceErrorIdentifierPrefixRule.php

```diff
@@ -0,0 +1,70 @@
+<?php declare(strict_types = 1);
+
+namespace ShipMonk\PHPStan\Rule;
```

Comment:
I would prefer using different namespace, e.g.

```suggestion
namespace ShipMonk\PHPStanDev\Rule;
```